tservice/sys/wrapper.h

```diff
@@ -0,0 +1,3 @@
+#include <sgx_tseal.h>
```

Comment:
Some functions exported by `tservice.a` are declared in `sgx_secure_align_api.h` (specifically `sgx_aligned_malloc`, `sgx_aligned_free`, and `sgx_get_aligned_ptr`. I believe this header should be included in the wrapper for that reason.

Reply:
added missing header